boilerplates/sentry/files/$vite.config.ts.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  addVitePlugin,
+  loadAsMagicast,
+  deepMergeObject,
+  generateCode,
+  builders,
+  type TransformerProps,
+} from "@batijs/core";
+
+export default async function getViteConfig(props: TransformerProps) {
+  const mod = await loadAsMagicast(props);
+
+  if (props.meta.BATI.has("sentry")) {
+    // load app level environment variables from `.env` file
+
+    // add dotenv to load environment variables from `.env` file
+    mod.imports.$add({
+      from: "dotenv",
+      imported: "config",
+    });
+
+    const e = builders.functionCall("config");
+    const c = generateCode(e).code;
+    //@ts-ignore
+    mod.$ast.body.splice(mod.$ast.body.length - 1, 0, c);
+
+    addVitePlugin(mod, {
+      from: "@sentry/vite-plugin",
+      constructor: "sentryVitePlugin",
+      imported: "sentryVitePlugin",
+      options: {
+        org: builders.raw("process.env.SENTRY_ORG"),
+        project: builders.raw("process.env.SENTRY_PROJECT"),
+        authToken: builders.raw("process.env.SENTRY_AUTH_TOKEN"),
```

Comment:
We could probably use `import.meta.env` instead

Reply:
if we use `import.meta.env` we would have to prefix the names with `PUBLIC_` which would leak `SENTRY_AUTH_TOKEN` to the APP. 

I placed all this values in `.env` to support the follow use cases:
* Sentry Browser which only need `PUBLIC_ENV__SENTRY_DSN`
* SentryVitePlugin which does the adding of the debug id, release and uploads the source map files - needs the rest (this needs to be done for the browser code and the server code if Sentry Server is used)
* Sentry Server (e.g. Sentry for AWS) uses `SENTRY_DSN` which can be the same as `PUBLIC_ENV__SENTRY_DSN` but this is optional.

I am too not happy to include `dotenv` but did not found any other way except manual loading `.env` and parsing. I open to any solution which removes the need to use `dotenv`